x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -23,35 +37,23 @@ export const AGENT_NAMES: AgentName[] = [
   'python',
   'ruby',
   'rum-js',
+  ...OPEN_TELEMETRY_AGENT_NAMES,
 ];
 
-export function isAgentName(agentName: string): agentName is AgentName {
-  return AGENT_NAMES.includes(agentName as AgentName);
-}
-
-export const RUM_AGENTS = ['js-base', 'rum-js'];
+export const RUM_AGENT_NAMES: AgentName[] = [
+  'js-base',
+  'rum-js',
+  'opentelemetry/webjs',
+];
 
 export function isRumAgentName(
   agentName?: string
-): agentName is 'js-base' | 'rum-js' {
-  return RUM_AGENTS.includes(agentName!);
+): agentName is 'js-base' | 'rum-js' | 'opentelemetry/webjs' {
+  return RUM_AGENT_NAMES.includes(agentName! as AgentName);
 }
 
 export function isJavaAgentName(
   agentName: string | undefined
 ): agentName is 'java' {
   return agentName === 'java';
 }
-
-/**
- * "Normalizes" and agent name by:
- *
- * * Converting to lowercase
- * * Converting "rum-js" to "js-base"
- *
- * This helps dealing with some older agent versions
- */
-export function getNormalizedAgentName(agentName?: string) {
```

Comment:
You got rid of this entirely? 👍

Reply:
Yes. It's functionality is replaced with `getAgentIconKey`.